Python/bytecodes.c

```diff
@@ -4359,17 +4362,20 @@ dummy_func(
         }
 
         // This is secretly a super-instruction
-        op(_CALL_LIST_APPEND, (callable, self, arg -- )) {
+        op(_CALL_LIST_APPEND, (callable, self, arg -- s, c)) {
```

Comment:
Can you please order it in the same order as the input args? So
```suggestion
        op(_CALL_LIST_APPEND, (callable, self, arg -- c, s)) {
```